### PR TITLE
[MediaPlayer] add API to set/get replaygain

### PR DESCRIPTION
**Pull request by kqjy777**
### API Changes ###

If you have the ACR for changing APIs, put the link of the ACR:
 - ACR: 

If you don't have the ACR, List all API changes here (or just put None), example:

Added:
 - bool Replaygain { set; get; } // Property

### CODE REVIEW

File: src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs [hunk 1]
```diff
@@ -523,6 +523,31 @@ public bool IsAudioOnly
...
 
+        /// <summary>
+        /// Gets or sets the player's replaygain state.
+        /// </summary>
+        /// <value>If the replaygain status is true, replaygain is applied (if contents has a replaygain tag);
+        /// otherwise, the replaygain isn't affected by tag and properties.</value>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <since_tizen> 5 </since_tizen>
+        public bool Replaygain
```
Comment (hsgwon) [inline src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs]: How about change this to ```ReplayGain```? I refered below.
- https://en.wikipedia.org/wiki/ReplayGain

File: src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs [hunk 2]
```diff
@@ -523,6 +523,31 @@ public bool IsAudioOnly
...
+        {
+            get
+            {
+                ValidateNotDisposed();
+                NativePlayer.IsReplaygain(Handle, out var value).
```
Comment (hsgwon) [inline src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs]: Same above comment. IsReplaygain -> isReplayGain

File: src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs [hunk 3]
```diff
@@ -523,6 +523,31 @@ public bool IsAudioOnly
...
+                    ThrowIfFailed(this, "Failed to get the replaygain of the player");
+                return value;
+            }
+            set
+            {
+                ValidateNotDisposed();
+                NativePlayer.SetReplaygain(Handle, value).
```
Comment (hsgwon) [inline src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs]: Same above comment. ```SetReplaygain``` -> ```SetReplayGain```

File: src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs (hunk 1 above)
Comment (kqjy777) [reply src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs]: ok i'll fix it

File: src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs (hunk 2 above)
Comment (kqjy777) [reply src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs]: ok i'll fix it

File: src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs (hunk 3 above)
Comment (kqjy777) [reply src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs]: ok i'll fix it

**Comment by WonyoungChoi** [comment]
http://suprem.sec.samsung.net/jira/projects/TCSACR/issues/TCSACR-154

**Pull request merged**